DXMainClient/Online/ChannelUser.cs

```diff
@@ -13,5 +13,6 @@ public ChannelUser(IRCUser ircUser)
         public IRCUser IRCUser { get; private set; }
 
         public bool IsAdmin { get; set; }
+        public bool IsFriend { get; set; }
```

Comment:
This property should be moved to IRCUser. After all, if a user is a friend of yours on one channel, they'll also be your friend on another channel.